conf/conf.yaml

```diff
@@ -11,6 +11,12 @@ etcd:
 
     verify: true                  # whether to verify the etcd endpoint certificate when setup a TLS connection to etcd,
     # the default value is true, e.g. the certificate will be verified strictly.
+log:
+  level: info
+  path: apisix-seed.log           # path is the file to write logs to.  Backup log files will be retained in the same directory
+  maxage: 24h                     # maxage is the maximum number of days to retain old log files based on the timestamp encoded in their filename
+  #maxsize: 102400                # maxsize is the maximum size in megabytes of the log file before it gets rotated. It defaults to 100mb
```

Comment:
Why do you comment it, unlike other fields? And the comment says the default value is 100, but 102400 is given in the default conf...